cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -45,6 +45,8 @@ class Global {
 
 def pluginGroup = 'cleaners'
 def config = new ConfigSlurper().parse(new File(ctx.artifactoryHome.haAwareEtcDir, PROPERTIES_FILE_PATH).toURL())
+def regex = ~/-\d+\.\d+\.\d+/
```

Comment:
Could you give as well an example what this regex is used for (semantic versioning).

Reply:
This regex will keep all artifacts that contains 3 digits version in its name after "-" char. So for example:
art-2.0.0.tgz - will be kept
alpine-20.21.0.tar.gz - will be kept 
alpine-2.tar.gz - will be deleted
alpine-2.0.tar.gz - will be deleted

Reply:
how does it handle navkit releases like `blah-release-bin-2.0.94-17.1.zip`?

Reply:
@wiorka:
`Can't delete docker-local:blah-release-bin-2.0.94-17.1.zip (user admin has no delete permissions), 3/5 total 509116928 bytes`

Reply:
This is just an example regex.